src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -221,6 +221,22 @@ class FunSpecTest {
       |fun foo(string: kotlin.String, nodoc: kotlin.Boolean): kotlin.String = "foo"""".trimMargin())
   }
 
+  @Test fun functionWithModifiedReturnKdoc() {
+    val funSpec = FunSpec.builder("foo")
+        .addParameter(ParameterSpec.builder("nodoc", Boolean::class).build())
```

Comment:
supernit: use the overload that accepts `name` and `type`, no need to create `ParameterSpec` manually here